pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -83,6 +85,65 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+
+  return (
+      <div
+          role="tabpanel"
+          hidden={props.value !== props.index}
+          id={`simple-tabpanel-${props.index}`}
+          aria-labelledby={`simple-tab-${props.index}`}
+      >
+        {props.value === props.index && (
+            <Box >
+              <Typography>{props.children}</Typography>
+            </Box>
+        )}
+      </div>
+  );
+}
+
+
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+export const ApplicationFormTabs: React.FC<ApplicationFormProps> = (props) => {
+  const [selectedTabIndex, setValue] = useState(0);
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+      <Box width={600}>
+        <Box >
+          <Tabs value={selectedTabIndex} onChange={handleChange} aria-label="basic tabs example">
+            <Tab label="Add manually" {...a11yProps(0)} />
+            <Tab label="Add from Git (Alpha)" {...a11yProps(1)} />
+          </Tabs>
+        </Box>
+        <TabPanel value={selectedTabIndex} index={0}>
+          <ApplicationForm {...props} />
+        </TabPanel>
+        <TabPanel value={selectedTabIndex} index={1}>
+          Comming soon...
+          {/** TODO: Show unregistered applications on the ADD FROM GIT tab */}
+        </TabPanel>
```

Comment:
```suggestion
        <TabPanel selected={selectedTabIndex === 0} index={0}>
          <ApplicationForm {...props} />
        </TabPanel>
        <TabPanel selected={selectedTabIndex === 1} index={1}>
          Comming soon...
          {/** TODO: Show unregistered applications on the ADD FROM GIT tab */}
        </TabPanel>
```
nits, I just feel like we can avoid this `value` name used here and use a bit more context provided variable `selected` since we use that value to determine should we show the tab children or not right? I usually make the component only focus on what to show and move the condition logic to the caller (in your component the part `props.value === props.index` is what I mean). When we do that way, we will have a single place where we can control display logic with the value we use (inside this `ApplicationFormTabs` component) that's easier for tracking and debugging. Besides, the props which need to be provided in order to use this `TabPanel` is easier to read too, it will be kind of
```
<TabPanel
   index={1}            // type number, index of the tab
   selected={true} // type bool, flag to control display based on is this Tab is selected or not
>
```

Reply:
> I usually make the component only focus on what to show and move the condition logic to the caller 

I feel the same way. Love it!

Reply:
Fixed at b9b20d7